hbase-noop-htrace/pom.xml

```diff
@@ -30,26 +30,29 @@
   <artifactId>hbase-noop-htrace</artifactId>
   <name>Apache HBase Drop-in noop HTrace replacement</name>
   <description>
-    Implmeents the HTrace APIs with noops and nonsense aimed at a) removing CVE flagged transitive
+    Implements the HTrace APIs with noops and nonsense aimed at a) removing CVE flagged transitive
     dependencies and b) favoring the JIT optimizing it away.
   </description>
   <properties>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <!-- This library is meant to work with whatever HBase and Hadoop combos we still list
          as recommended. The version here is what we'll test out of the box if you run mvn test
       -->
-    <hbase.stable.version>2.4.8</hbase.stable.version>
+    <hbase.stable.version>2.4.17</hbase.stable.version>
     <!-- should match the slf4j-api version used in the above hbase -->
-    <slf4j.version>1.7.30</slf4j.version>
+    <slf4j.version>1.7.33</slf4j.version>
     <!-- These maybe need to match the main repo -->
     <maven.checkstyle.version>3.1.0</maven.checkstyle.version>
     <checkstyle.version>8.29</checkstyle.version>
+    <!-- Added mockito-all as otherwise we get java.lang.NoClassDefFoundError:
```

Comment:
Hi @Apache9 I have retained this change, although this should not be needed when compiling with hbase 2.4.x but would be use if compiling with 2.5.x. Let me know if you want to me to drop this.